Update README.md (#37)
UtrechtUniversity/animal-sounds: README.md

- Jelle Treep (h.j.treep@uu.nl)

### Dataset description

> The audio dataset collected in tropical forests of the Congo Basinand, and Semi-natural Chimanze enclosures in Cameroon. The recordings are taken at three locations:

Reviewer: ```suggestion
The dataset for this project contains recordings in `.wav` format at 1 minute length and at a sample rate of 48000 samples/second. The recordings are taken at three locations in (or close to) the tropical rainforest of Cameroon and Congo:
```

- Chimpanze sanctuary - Congo
- Natural forest - Congo
- Semi-natural Chimanze enclosures - Cameron 

|Dataset| # Chimpanze samples | # Background samples |
| --- | --- | --- |
| Sanctuary | 17.921 | 74.163 | 
| Synthetic | 68.757 | 97.149 | 

> The dataset for this project contains recordings in `.wav` format at 1 minute length and at a sample rate of 48000 samples/second.

Reviewer: ```suggestion
```

> | Synthetic | 68.757 | 97.149 | 

Reviewer: explain synthetic

- Jelle Treep (h.j.treep@uu.nl)

### Dataset description
The initial dataset for this project contains recordings in `.wav` format at 1 minute length and at a sample rate of 48000 samples/second. The recordings are taken at three locations in (or close to) the tropical rainforest of Cameroon and Congo:

- Chimpanze sanctuary - Congo
- Natural forest - Congo
- Semi-natural Chimanze enclosures - Cameron 

> In a data augmentation process we also generated synthetic data from Congo dataset. The Chimpanze sanctuary and Natural forest are used for training and optimizing the classifiers. The Semi-natural Chimpanze recordings are used as an independent evaulation of the classifiers.

Reviewer: ```suggestion
```
